app/policies/repository_policy.rb

```diff
@@ -1,4 +1,15 @@
 class RepositoryPolicy
+  attr_reader :user, :repository
```

Comment:
t you're not using them, are you? If you remove this file, tests are not passing?